content/en/docs/_index.md

```diff
@@ -5,22 +5,21 @@ description: "Everything you need to know about how the documentation is organiz
 
 # Welcome
 
-Welcome to the [Helm](https://helm.sh/) documentation. Helm is the package
-manager for Kubernetes, and you can read detailed background information in
-the [CNCF Helm Project Journey report](https://www.cncf.io/cncf-helm-project-journey/).
+Welcome to the [Helm](https://helm.sh/) documentation. Helm is the package manager for Kubernetes,
+and you can read detailed background information in the [CNCF Helm Project Journey
+report](https://www.cncf.io/cncf-helm-project-journey/).
 
-# How the documentation is organized
+## How the documentation is organized
```

Comment:
I accidentally introduced a second top-level heading in https://github.com/helm/helm-www/pull/703 and it's great that you caught it here!